mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelFactory.kt

```diff
@@ -78,23 +83,29 @@ sealed class ViewModelContext {
  * reference is available when an activity scoped ViewModel is first created, during process restoration, activity scoped ViewModels will be created
  * _without_ a fragment reference, so it is only safe to reference the activity.
  */
-class ActivityViewModelContext(
+data class ActivityViewModelContext(
     override val activity: FragmentActivity,
     override val args: Any?
-) : ViewModelContext()
+) : ViewModelContext() {
+    override val owner = activity
```

Comment:
you can use `get() = `for all of the assignments you added in this file to avoid object allocations. We don't really need to store the separately, and is nice to minimize memory that our library uses for people